Update iOS URL Scheme Handling Documentation for Swift (#49)
react-native-google-signin/docs: docs/setting-up/ios.md

### Optional: modify your app to respond to the URL scheme

This is only required if you have multiple listeners for `openURL` - for instance if you have both Google and Facebook OAuth.

Because only one `openURL` method can be defined, if you have multiple listeners for `openURL`, you must combine them into a single function as shown below:

> #### For React Native >= 0.77.0

Reviewer: ```suggestion
#### For AppDelegate written in Swift
```

> If you're using React Native 0.77.0 or higher, you'll need to:

Reviewer: ```suggestion
If your AppDelegate a Swift file (the default in React Native 0.77.0 or higher), you'll need to:
```

> 1. Add the following import to your project's [bridging header](<(https://developer.apple.com/documentation/swift/importing-objective-c-into-swift#Import-Code-Within-an-App-Target)>) file (usually `ios/YourProject-Bridging-Header.h`):

Reviewer: ```suggestion
1. Add the following import to your project's [bridging header](https://developer.apple.com/documentation/swift/importing-objective-c-into-swift#Import-Code-Within-an-App-Target) file (usually `ios/YourProject-Bridging-Header.h`):
```

```objc
// …

// ⬇️ Add this import
#import <GoogleSignIn/GoogleSignIn.h>
```

2. Modify your `AppDelegate.swift` file:

```swift
// …

@main
class AppDelegate: RCTAppDelegate {
  // …

  // ⬇️ Add this method
  override func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
    // Add any other URL handlers you're using (e.g. Facebook SDK)
    return ApplicationDelegate.shared.application(app, open: url, options: options) ||
           GIDSignIn.sharedInstance.handle(url)
  }

}
```

> #### For React Native < 0.77.0

Reviewer: ```suggestion
#### For AppDelegate written in Objective-C
```

> For older versions of React Native, modify your `AppDelegate.m` file:

Reviewer: ```suggestion
For AppDelegate written in Objective-C (the default prior to React Native 0.77), modify your `AppDelegate.m` file:
```
